cmd/promtool/unittest.go

```diff
@@ -91,7 +101,8 @@ func ruleUnitTest(filename string, queryOpts promqltest.LazyLoaderOpts, run *reg
 	}
 
 	evalInterval := time.Duration(unitTestInp.EvaluationInterval)
-
+	currentTime := time.Now().Format("2006-01-02T15:04:05")
+	ts.Settime(currentTime)
 	// Giving number for groups mentioned in the file for ordering.
 	// Lower number group should be evaluated before higher number group.
 	groupOrderMap := make(map[string]int)
```

Comment:
There's a new test here on line 110 compared to the original PR. I think we should create a `ts.Case("unittest_check_ordering")` and record `Fail` when it returns error.